GetUserCommand ui_command
---
GetResult sys_result
---